can/interfaces/slcan.py

```diff
@@ -0,0 +1,147 @@
+"""
+Interface for slcan compatible interfaces (win32/linux).
+(Linux could use slcand/socketcan also).
+"""
+from __future__ import absolute_import
+
+import serial
+import io
+import time
+import logging
+
+from can import CanError, BusABC, Message
+
+
+logger = logging.getLogger(__name__)
+
+class slcanBus(BusABC):
+    """slcan interface"""
+
+    def write(self, str):
+        if not str.endswith("\r"):
+            str += "\r"
+        self.serialPort.write(str.decode())
+        self.serialPort.flush()
+
+    def open(self):
+        self.write("O")
+
+    def close(self):
+        self.write("C")
+
+
+    def __init__(self, channel, ttyBaudrate=115200, timeout=1, bitrate=None, poll_interval=0.01 , **kwargs):
+        """
+        :param string channel:
+            port of underlying serial or usb device (e.g. /dev/ttyUSB0, COM8, ...)
+        :param int ttyBaudrate:
+            baudrate of underlying serial or usb device
+        :param int bitrate:
+            Bitrate in bits/s
+        :param float poll_interval:
+            Poll interval in seconds when reading messages
+        :param float timeout
+            timeout in seconds when reading message
+        """
+
+
+        if channel == '':
+            raise TypeError("Must specify a serial port.")
+        if '@' in channel:
+            (channel, ttyBaudrate) = channel.split('@')
+
+        self.poll_interval = poll_interval
```

Comment:
This is not used.

Reply:
yes, you are right. 
poll_intervall is a copy-paste artefact
I'll remove